### DISCUSSION TITLE
Use case general properties

### DISCUSSION THREAD

**Discussion by Sahin-Albayram** [bounswe/bounswe2023group2#95] [General]
This discussion concerns @mervegrbz and @egecans. I suggest that our project should have three main parts:
- Map: Where users can view almost everything on map
- Dashboard: Where users can view things on dashboard. It eases the searching utility.
- Profiles: Where users can view both their own and other users' profiles. 

Anything other that group beneath these three parts. For example activities are part of both dashboard and map.

**Comment by bgmrsln** [comment]
How about creating three web page mock ups for each. The one for profile can also be named Account :
<img width="970" alt="image" src="https://user-images.githubusercontent.com/53114791/230721599-80db7dff-a438-485c-955f-8930e01e54c6.png">
I guess we should also differentiate this mockup for different views.

**Reply by bgmrsln** [reply]
We can call is profile page and call sub page as user settings as well

**Comment by rburaksaritas** [comment]
Where will the creating activity functionality be resided? Map or Dashboard? (and Profile i guess)
